webapp/packages/plugin-codemirror6/src/ReactCodemirror.tsx

```diff
@@ -126,6 +126,12 @@ export const ReactCodemirror = observer<IReactCodeMirrorProps, IEditorRef>(
 
         editorView.dom.addEventListener('keydown', event => {
           const newEvent = new KeyboardEvent('keydown', event);
+
+          // we handle undo/redo on the CaptureView level
+          if ((newEvent.metaKey || newEvent.ctrlKey) && ['z', 'y'].includes(newEvent.key.toLowerCase())) {
+            return;
+          }
+
```

Comment:
this is for purpose, cuz we can have different editors (for example VQB)
we have our own changes history implementation, please check `SqlDataSourceHistory`

Probably we have a collision with our own implementation and codemirror (we need to disable it for codemirror and use only ours)